src/components/withCurrentUserPersonalDetails.tsx

```diff
@@ -4,7 +4,7 @@ import useCurrentUserPersonalDetails from '@hooks/useCurrentUserPersonalDetails'
 import getComponentDisplayName from '@libs/getComponentDisplayName';
 import type {PersonalDetails} from '@src/types/onyx';
 
-type CurrentUserPersonalDetails = PersonalDetails | Record<string, never>;
+type CurrentUserPersonalDetails = PersonalDetails;
```

Comment:
Maybe let's just remove this `CurrentUserPersonalDetails` type altogether and use `PersonalDetails` instead?